plugin/pulpcore/plugin/stages/models.py

```diff
@@ -0,0 +1,66 @@
+from gettext import gettext as _
+
+
+class DeclarativeArtifact:
+    """
+    Relates an Artifact, how to download it, and its relative_path for publishing.
+
+    This is used by the Stages API stages to determine if an Artifact is already present and ensure
+    Pulp can download it in the future. The `artifact` can be either saved or unsaved. If unsaved,
+    the `artifact` attributes may be incomplete because not all digest information can be computed
+    until the Artifact is downloaded.
+
+    Attributes:
```

Comment:
This is causing the generated docs to not call these parameters, but variables instead. It renders weird: 
![declarativeartifactdocs](https://user-images.githubusercontent.com/294339/43269593-2cc828ba-90c1-11e8-9b82-e2d5c41133f2.png)

Reply:
I'm fixing this in the next push by using Sphinx's `:no_members:`